azure_functions_worker/dispatcher.py

```diff
@@ -700,6 +743,49 @@ def gen(resp_queue):
             error_logger.exception('unhandled error in gRPC thread')
             raise
 
+    def _process_indexed_function(self, indexed_functions: List[Function]):
```

Comment:
I am thinking if `loader.py` should be the place for all the code here?

Reply:
I cannot review this as `codecov` is adding a log of messages. Could you please check if the lines are covered?

<img width="528" alt="image" src="https://user-images.githubusercontent.com/1047040/159215960-07b5c730-1913-4dec-aef1-1c0d396cef83.png">